lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -500,6 +502,20 @@ describe('BasicLTILaunchApp', () => {
       renderLTILaunchApp();
       assert.isTrue(fakeApiCall.notCalled);
     });
+
+    it('reports a submission if VitalSource configuration is present, even if `contentUrl` not set', async () => {
+      // "Un-set" eventual contentUrl
```

Comment:
The internal state is `contentURL`. Again, I think the code speaks for itself so I'd just remove the comment.